src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/PersonEntity.kt

```diff
@@ -148,6 +148,10 @@ class PersonEntity(
         nationalInsuranceNumber = person.nationalInsuranceNumber,
         sourceSystem = person.sourceSystemType,
       )
+      val personAliases = PersonAliasEntity.fromList(person.personAliases)
+      personAliases.forEach { personAliasEntity -> personAliasEntity.person = personEntity }
+      personEntity.aliases.addAll(personAliases)
+      return personEntity
       return personEntity
```

Comment:
2 returns here.

Reply:
surprised that detekt or the linter did not find this. Good spot, thanks